default.nix

```diff
@@ -280,52 +294,30 @@ let
         name = "plutus-development";
         contents =
           let runtimeGhc =
-                haskellPackages.ghcWithPackages (ps: [
-                  haskellPackages.language-plutus-core
-                  haskellPackages.plutus-emulator
-                  haskellPackages.plutus-wallet-api
-                  haskellPackages.plutus-tx
-                  haskellPackages.plutus-tx-plugin
-                  haskellPackages.plutus-use-cases
-                  haskellPackages.plutus-ir
-                  haskellPackages.plutus-contract
```

Comment:
It makes no difference, I think: in both cases they end up being drawn from the package set that you took the `ghcWithPackages` from. I think the only real reason for the `ps` argument is so you can pass around `ghcWithPackages` independently without the package set. But since it's there, I thought we should use it.